test/promap.js

```diff
@@ -0,0 +1,39 @@
+var assert = require('assert');
+
+var R = require('../source');
+var eq = require('./shared/eq');
+var Id = require('./shared/Id');
+
+
+describe.only('promap', function() {
+  var fromCharCode = String.fromCharCode;
+  function charCodeAt(s) { return s.charCodeAt(); }
+  function Costar(run) { return { promap: promap, run: run };
+    function promap(f, g) {
+      return Costar(R.pipe(R.map(f), run, g));
+    }
+  }
```

Comment:
Could we reformat this please?  Either something like this:

```js
  function Costar(run) { 
    return { promap: promap, run: run };

    function promap(f, g) {
      return Costar(R.pipe(R.map(f), run, g));
    }
  }
```

or like this:

```js
  function Costar(run) { 
    return { 
      promap: function promap(f, g) {
        return Costar(R.pipe(R.map(f), run, g));
      }, 
      run: run 
    };
  }
```

Reply:
hehe, fair - I knew I was taking a gamble with that one 😛